src/backend/cuda/kernel/memcopy.hpp

```diff
@@ -86,11 +87,10 @@ void memcopy(T *out, const dim_t *ostrides, const T *in, const dim_t *idims,
     POST_LAUNCH_CHECK();
 }
 
-////////////////////////////// BEGIN - templated help functions for copy_kernel
-///////////////////////////////////
+///////////// BEGIN - templated help functions for copy_kernel /////////////////
```

Comment:
I thought we are removing these block comments whenever we are editing surrounding code.